package.json

```diff
@@ -20,6 +20,7 @@
     "test:all": "npm run test && npm run test:integration",
     "test:all:redis": "npm run test && npm run test:integration:redis",
     "test:all:redis:debug": "npm run test && npm run test:integration:redis:debug",
+    "doc": "lerna run doc",
```

Comment:
Can we skip adding this? Just one less thing to consider when we remove lerna.

Reply:
I guess we could, I was imagining the docs would be updated by the designated "release engineer", I was going to update the release doc to state npm run doc should kick off after npm run build, I put this in here for convenience.

I can either remove it or link this directly to the command in the generator package.json (or just leave it here for now).
I don't think it's a big deal either way, but happy to go which ever direction you like.